src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -225,21 +231,25 @@ export default class CancerStudySelector extends React.Component<ICancerStudySel
                             {!!(!this.store.cancerTypes.isPending && !this.store.cancerStudies.isPending) && (
                                 <Observer>
                                     {() => {
+                                    const studyLimitReached = this.store.selectableSelectedStudyIds.length > 50;
+                                    const multipleReferenceGenomes = (!this.checkForSingleReferenceGenome);
+                                    let tooltipMessage = <span>Open summary of selected studies in a new window.</span>;
+                                    if (studyLimitReached) {
```

Comment:
i think these two ifs should be if/else and probably studyLimitReached should show first.

Reply:
agreed